tests/test_runtime.py

```diff
@@ -551,23 +550,29 @@ def test_too_many_param(self, token, tol, shots):
 
         def vqe_circuit(params):
             qml.RX(params[0], wires=0)
-            qml.RX(params[1], wires=0)
+            qml.RY(params[1], wires=0)
 
         coeffs = [1, 1]
         obs = [qml.PauliX(0), qml.PauliZ(0)]
 
         hamiltonian = qml.Hamiltonian(coeffs, obs)
         program_id = upload_vqe_runner(hub="ibm-q-startup", group="xanadu", project="reservations")
 
-        job = vqe_runner(
-            program_id=program_id,
-            backend="ibmq_qasm_simulator",
-            hamiltonian=hamiltonian,
-            ansatz=vqe_circuit,
-            x0=[3.97507603, 3.00854038, 3.55637849],
-            shots=shots,
-            optimizer_config={"maxiter": 10},
-            kwargs={"hub": "ibm-q-startup", "group": "ibm-q-startup", "project": "reservations"},
+        with pytest.warns(UserWarning) as record:
```

Comment:
Worth adding a note on this to the PR description.

Reply:
Yes I am adding it!